### PR TITLE
feat: Added deep link(MetaMask) for mobile browser

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* Added deep link(MetaMask) for mobile browser

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

### CODE REVIEW

File: src/hooks/useConnectWallet.ts [hunk 1]
```diff
@@ -127,16 +134,21 @@ export const useConnectWallet = () => {
...
   watch(
     [isConnectedNetwork],
-    () => {
+    async () => {
       const address = localStorage.getItem(SELECTED_ADDRESS);
-      if (!address || !isConnectedNetwork.value) return;
-      // Memo: wait for updating the chain id from the initial state 592 (to pass the `setupNetwork` function)
-      setTimeout(async () => {
-        if (address === 'Ethereum Extension') {
+      const isMetaMaskDeepLink = window.location.hash === deepLinkPath.metamask;
```
Comment (hoonsubin) [inline src/hooks/useConnectWallet.ts:139]: Can you add deep links for other mobile wallets too? Like Math wallet and Nova wallet

Comment (impelcrypto) [reply src/hooks/useConnectWallet.ts:172]: I've checked with MathWallet and confirmed MathWallet doesn't have this feature at this moment.

**Review by hoonsubin — APPROVED** [review]

**Comment by impelcrypto** [comment]
Hi @sirius651 @bobo-k2 
Could you take a look at this PR, please?

**Review by sirius651 — APPROVED** [review]
LGTM

**Pull request merged**